src/snovault/elasticsearch/searches/queries.py

```diff
@@ -850,6 +850,9 @@ def add_sort(self):
                 *sort_by
             )
 
+    def add_exact_counting(self):
```

Comment:
Nice! You'll have to add an `add_exact_counting` test to continue our 100% coverage of this code. Can see https://github.com/ENCODE-DCC/snovault/blob/79b5524ce9b9d7f7d32a1dbc143230f7efc5873a/src/snovault/tests/test_searches_queries.py#L2905 etc. for pattern.

Reply:
https://github.com/ENCODE-DCC/snovault/blob/2b86c6e7820979de77989c54a12d57fa03298cb2/src/snovault/tests/test_searches_queries.py#L2903